functions/APIs/farms.js

```diff
@@ -103,6 +103,30 @@ exports.deleteFarm = (request, response) => {
       if (!doc.exists) {
         return response.status(404).json({ error: "Farm Object not found" });
       }
+      db.collection("surplus")
```

Comment:
Can you update the function comments to reflect that more than a farm may be deleted?
e.g. Delete a specific farm and any surplus or deals associated with it

Please address throughout. (and while you're there, can you fix up the function comments to use proper style? https://google.github.io/styleguide/jsguide.html#jsdoc-general-form but the params aren't necessary)